src/app/api/categories/route.ts

```
@@ -0,0 +1,27 @@
+            },
+            cache: "no-store",
+        });
+
+        if (!res.ok) {
+            return NextResponse.json({ message: "Failed to fetch categories" }, { status: 500 });
+        }
+
+        const data: CategoriesResponse = await res.json();
+        return NextResponse.json(data);
+    } catch (error) {
+        console.error(error);
```

ESLint

[error] Unexpected console statement.

(no-console)